File: martianrun/core/src/com/gamestudio24/martianrun/box2d/GroundUserData.java
package com.gamestudio24.martianrun.box2d;

import com.gamestudio24.martianrun.enums.UserDataType;

public class GroundUserData extends UserData {

    public GroundUserData(float width, float height) {
        super(width, height);
        userDataType = UserDataType.GROUND;
    }
}

File: martianrun/core/src/com/gamestudio24/martianrun/box2d/GroundUserData.kt
package com.gamestudio24.martianrun.box2d

import com.gamestudio24.martianrun.enums.UserDataType

class GroundUserData(width: Float, height: Float) : UserData(width, height) {
    init {
        userDataType = UserDataType.GROUND
    }
}